dartagnan/src/test/java/com/dat3m/dartagnan/others/miscellaneous/AnalysisTest.java

```diff
@@ -101,7 +101,7 @@ private void reachingDefinitionMustOverride(ReachingDefinitionsAnalysis.Method m
         Configuration config = Configuration.builder().setOption(REACHING_DEFINITIONS_METHOD, method.name()).build();
         Context context = Context.create();
         context.register(BranchEquivalence.class, BranchEquivalence.fromConfig(program, config));
-        context.register(ExecutionAnalysis.class, ExecutionAnalysis.fromConfig(program, ProgressModel.FAIR, context, config));
+        context.register(ExecutionAnalysis.class, ExecutionAnalysis.fromConfig(program, ProgressModel.uniform(ProgressModel.FAIR), context, config));
```

Comment:
Why not `ProgressModel::defaultHierarchy`?

Reply:
The original tests were explicitly stating FAIR. Since technically the default progress model might change in the future (though unlikely to ever happen), it is better to have the tests be independent of the default settings.